SabreTools.Hashing.Test/HashToolTests.cs

```diff
@@ -125,5 +125,17 @@ public void GetStreamHashesAndSizeTest()
             TestHelper.ValidateSize(actualSize);
             TestHelper.ValidateHashes(hashDict);
         }
+        
+        [Fact]
+        public void GetFuzzyCompareTest()
```

Comment:
Looking at the `TestHelper` code you implemented, most of that can be directly inlined into this test method instead with little issue.